puzzles/templates/puzzles_table.html

```diff
@@ -119,6 +125,17 @@
             { "type": "name-input", "targets": [0] },
             { "type": "select-input", "targets": [2] },
             { "type": "tag-input", "targets": [5] },
+        ],
+        // Manual sizing as an optimization: saves ~1s on a 300 puzzle table
+        "autoWidth": false,
+        "columns": [
+            { "width": "30%" },
+            { "width": "17.5%" },
+            { "width": "160px" },
+            { "width": "60px" },
+            { "width": "60px" },
```

Comment:
I hope these new sizes work ok on various devices and browser window sizes.

Reply:
It looks ok at 960 wide, 1280 wide and 1920 wide, so I think it's fine.